examples/styling/triangle.svg

```diff
@@ -0,0 +1 @@
+<svg version="1.1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 15 15"><path d="M7.538 2c-.294 0-.488.177-.615.385l-5.846 9.538C1 12 1 12.153 1 12.308c0 .538.385.692.692.692h11.616c.384 0 .692-.154.692-.692 0-.154 0-.231-.077-.385l-5.77-9.538C8.029 2.177 7.789 2 7.539 2z"/></svg>
```

Comment:
Why is this SVG added here? I think `triangle.svg` is a built-in

Reply:
Yes, I accidentally added this file. Removing it.